src/ui/sass/modules/_SortOptions.scss

```diff
@@ -45,6 +45,18 @@
     margin-left: 5px;
   }
 
+  &-optionSelector {
+    &:focus {
+      outline: .0625rem auto black;
```

Comment:
could you double check whether the other focus styling uses 1px or .0625 rem? (I haven't checked myself)

Reply:
The ask was .0625rem, but the other focus styling uses 1px. I can double check with rose